doc-content/enterprise-only/getting-started/filtering-process-instances-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='filtering-process-instances-proc']
```

Comment:
Not modularized correctly. Needs to be broken up and intro/context statements added to the procs. This module should be the con, with a con-appropriate name (not a gerund), and the floating sub-headings (never okay in a proc), need to be proc modules included in the assembly after this con, and each need an intro/context statement.

For example:
```
[id='process-instances-filters-con']
= Process instance filters
{Central} now provides you ....

<NEW MODULE FILE>
[id='process-instances-filters-proc']
= Filtering process instances using default filters
<Intro/context statement>

.Procedure
....

<NEW MODULE FILE>
[id='process-instances-filters-advanced-proc']
= Filtering process instances using advanced filters
<Intro/context statement>

.Procedure
....
```